### PR TITLE
6 adding logger

### PR DESCRIPTION
**Pull request by AnaLanda**

### CODE REVIEW

**Review by DmGrt — APPROVED** [review]
Good job!

File: src/main/resources/log4j.properties [hunk 1]
```diff
@@ -0,0 +1,16 @@
+log4j.rootLogger=INFO, stdout, file
+log4j.logger.org.hibernate=INFO
+log4j.logger.org.hibernate.SQL=DEBUG
+log4j.logger.org.hibernate.type.descriptor.sql=INFO
```
Comment (KseniiaMakarova) [inline src/main/resources/log4j.properties]: do you want to save hibernate logs intentionally?

File: src/main/java/com/cinema/Main.java [hunk 2]
```diff
@@ -57,19 +59,19 @@ public static void main(String[] args) throws AuthenticationException, Interrupt
...
                 (AutheticationService) injector.getInstance(AutheticationService.class);
         User user1 = new User();
         user1.setEmail("user@gmail.com");
         user1.setPassword("password");
-        System.out.println("User1 has been registered: "
+        log.info("User1 has been registered: "
                 + autheticationService.register(user1.getEmail(), user1.getPassword()));
-        System.out.println("User1 has logged in: "
+        log.info("User1 has logged in: "
                 + autheticationService.login(user1.getEmail(), user1.getPassword()));
```
Comment (KseniiaMakarova) [inline src/main/java/com/cinema/Main.java]: let's wrap call to `login()` method in try-catch and add log.warn with the informative message in `catch` block

File: src/main/java/com/cinema/dao/impl/ShoppingCartDaoImpl.java [hunk 3]
```diff
@@ -6,21 +6,26 @@
...
 
 @Dao
 public class ShoppingCartDaoImpl implements ShoppingCartDao {
+    private static final Logger log = Logger.getLogger(ShoppingCartDaoImpl.class);
+
     @Override
     public ShoppingCart add(ShoppingCart shoppingCart) {
         Transaction transaction = null;
         Session session = null;
+        log.info("Trying to add the shopping cart " + shoppingCart + " to the DB.");
```
Comment (KseniiaMakarova) [inline src/main/java/com/cinema/dao/impl/ShoppingCartDaoImpl.java:22]: what about `update()` method?

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: src/main/resources/log4j.properties (hunk 1 above)
Comment (AnaLanda) [reply src/main/resources/log4j.properties]: Yes. But I can remove this)

Comment (KseniiaMakarova) [reply src/main/resources/log4j.properties]: it is ok if the logs remain clear

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**